### PR TITLE
scroll the metadata panel

### PR DESCRIPTION
**Pull request by akash1810**
# Before
scroll-before

# After
scroll-after

img

### CODE REVIEW

**Comment by akash1810** [comment]
> Does this solve the scrollbar position issue as well?

Yep.

**Comment by akash1810** [comment]
Just a thought - this panel could benefit from the accordion design pattern. At the moment, you have to scroll back up to see the rights, for example.

An accordion will mean rights (or anything high up) is a click away rather than a, possibly long, scroll.

Separate PR though!

File: kahuna/public/stylesheets/main.css [hunk 1]
```diff
@@ -1523,6 +1523,11 @@ FIXME: what to do with touch devices
     border-left: 1px solid #565656;
 }
 
+.image-details--scroll {
+    height: calc(100vh - 50px - 35px); /* viewport - top-bar - tab-bar */
```
Comment (akash1810) [inline kahuna/public/stylesheets/main.css]: Ugh, the tab-bar can be more than 35px (warning messages!)

Might move to CSS grid for this.

ugh

**Comment by paperboyo** [comment]
> accordion design pattern

👏 

Fixes https://github.com/guardian/grid/issues/1914.

**Pull request merged**